### ISSUE TITLE
Conformance statement clarity

### ISSUE THREAD

**Issue by rickj** [w3c/publ-a11y#206]
Section 3.7 of the current guidelines draft has many statements that use the word meet(s).  Some examples:

- This publication meets accepted accessibility standards.
- This publication meets minimum accessibility standards.
- This publication meets accepted accessibility standards and reports (EPUB Accessibility 1.1 and Web Accessibility Content Guidelines (WCAG) 2.1 Level AA.)

The use of the word 'meets' passes a judgement or a determination that is out of context, that I (as a vendor) do not want to declare about someone's content.  The note in 3.7 acknowledges that local legislation (e.g. the EAA) may have compliance requirements that could determine which conformance statement is true, a context that will not be available for me to evaluate.

In [issue #16](https://github.com/w3c/publ-a11y/issues/16) @avneeshsingh wrote an 'improved version' for creating conformance statements that were not abstract.  In that issue he stated:

> To report the accessibility conformance of a publication, metadata for accessibility conformance should be provided. It declares the accessibility standard or accessibility specifications to which the publication conforms. The value can be a URL or a code which identifies the accessibility standard or accessibility specifications.

Even in the last example above, where the specific standard claimed is referenced, we do not have clarity, as we have not defined what the word 'meets' means here.  That example cites two standards, but what does 'meets' mean for those two?  All fields are filled out for EPUB Accessibility 1.1? It claims to pass every success criteria for WCAG 2.1 AA?  Something else?

For conformance should we stick to just declaring what the specific claims are? (i.e This publication claims... )

**Comment by clapierre** [comment]
I see your point Rick, I think change "meets" to "claims to meet" might be appropriate here?

This publication claims to meet accepted accessibility standards.
This publication claims to meet minimum accessibility standards.
This publication claims to meet accepted accessibility standards and reports (EPUB Accessibility 1.1 and Web Accessibility Content Guidelines (WCAG) 2.1 Level AA.)

**Comment by GeorgeKerscher** [comment]
We clearly address this  where we insert the word claims in many places and in a section with  the word claims in theheading
